src/clr-angular/forms/datepicker/all.spec.ts

```diff
@@ -26,7 +26,7 @@ import LocaleHelperServiceSpecs from './providers/locale-helper.service.spec';
 import ViewManagerServiceSpecs from './providers/view-manager.service.spec';
 import YearpickerSpecs from './yearpicker.spec';
 
-describe('Datepicker', function() {
+fdescribe('Datepicker', function() {
```

Comment:
fdescribe

Reply:
Yup, was rebasing onto latest to resolve merge conflicts to address this.